src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -156,7 +156,7 @@
           </div>
 
           <!-- the detail comments section -->
-          <DetailsList
+          <Details-List
```

Comment:
Please change to "details-list" (eg, same as Vuetify components, eg, v-progress-circular).

Reply:
done, third times a charm!